eclair-core/src/main/scala/fr/acinq/eclair/wire/internal/channel/version1/ChannelCodecs1.scala

```diff
@@ -0,0 +1,266 @@
+package fr.acinq.eclair.wire.internal.channel.version1
+
+import fr.acinq.bitcoin.DeterministicWallet.{ExtendedPrivateKey, KeyPath}
+import fr.acinq.bitcoin.{ByteVector32, OutPoint, Transaction, TxOut}
+import fr.acinq.eclair.MilliSatoshi
+import fr.acinq.eclair.channel._
+import fr.acinq.eclair.crypto.ShaChain
+import fr.acinq.eclair.transactions.Transactions._
+import fr.acinq.eclair.transactions.{CommitmentSpec, DirectedHtlc, IncomingHtlc, OutgoingHtlc}
+import fr.acinq.eclair.wire.CommonCodecs.{bytes32, bytes64, mapCodec, publicKey, setCodec, varintoverflow, _}
+import fr.acinq.eclair.wire.LightningMessageCodecs.{channelAnnouncementCodec, channelUpdateCodec, closingSignedCodec, commitSigCodec, fundingCreatedCodec, fundingLockedCodec, fundingSignedCodec, lightningMessageCodec, shutdownCodec, updateAddHtlcCodec, _}
+import fr.acinq.eclair.wire.UpdateMessage
+import scodec.codecs.{bits, bool, byteAligned, bytes, discriminated, either, int64, listOfN, optional, provide, uint16, uint32, variableSizeBytesLong, _}
```

Comment:
nit:
```suggestion
import fr.acinq.eclair.wire.CommonCodecs._
import fr.acinq.eclair.wire.LightningMessageCodecs._
import fr.acinq.eclair.wire.UpdateMessage
import scodec.codecs._
```

Reply:
Weird, how come this isn't fixed by _optimize imports_?